src/core/src/FabricGeometryDefinition.cpp

```diff
@@ -46,6 +51,10 @@ bool FabricGeometryDefinition::operator==(const FabricGeometryDefinition& other)
         return false;
     }
 
+    if (_customVertexAttributes != other._customVertexAttributes) {
+        return false;
```

Comment:
We really don't need the branching in this operator? Why don't we just do...

```
    return _hasNormals == other._hasNormals && 
        _hasVertexColors == other._hasVertexColors && 
        _textcoordSetCount == other._texcoordSetCount &&
        _customVertexAttributes == other._customVertexAttributes
```

...instead? This suggestion seems clearer to me because it removes ambiguity that we want the all of these values to be equal.

Reply:
Done. Unfortunately clang-format doesn't place them nicely on separate lines.